### PR TITLE
Added new latency defs to Fping module, including return values

### PR DESCRIPTION
**Pull request by nckrse**
Added new latency defs to Fping module, including return values instead of using standard output

I wanted to use this module to return values to send fping metrics to remote destination (graphite in my case). I added two defs:

def \* latency: returns LOSS, MIN, AVG, MAX in an array 
0
13.7
15.5
17.3
(example code @ https://github.com/pythianreese/graphite-smokeping/blob/master/graphite-smokeping.rb)

def \* latency_simple: returns output (not modified)
4.2.2.2 : xmt/rcv/%loss = 6/6/0%, min/avg/max = 13.7/15.5/17.3

### CODE REVIEW

**Comment by penguinpowernz** [comment]
Thanks man!

**Comment by nckrse** [comment]
Np! You wouldn't mind pushing these changes to rubygems, would you? If any changes need to be made prior, let me know. I would like to use this as a gem dependency without bundler (using github source)

**Comment by penguinpowernz** [comment]
Yep, pushed that up for you now, version 0.2.1.  The code you submitted could use a bit of a tidying up but not many people use this gem so it's not urgent (see #2)
